src/codegen/sys/cargo_toml.rs

```diff
@@ -83,7 +83,7 @@ fn fill_in(root: &mut Table, env: &Env) {
 
     {
         let build_deps = upsert_table(root, "build-dependencies");
-        set_string(build_deps, "system-deps", "2.0");
+        set_string(build_deps, "system-deps", "3.0");
```

Comment:
Maybe we could just use `3` here? `3.x` will stay API compatible.

Reply:
I don't mind either way, but IIRC @GuillaumeGomez has opinions

Reply:
Also `3.0` and `3` are absolutely equivalent in how they're handled by cargo

Reply:
Then let's just use `3`. :)

Reply:
@gdesmott You update it?